source/cloud-native-platform/path-to-live/shutter.html.md.erb

```diff
@@ -43,3 +43,12 @@ Now everything is setup to render shutter page. Swap the DNS from Azure FrontDoo
 
 `shutter: true` will enable the shutter page and redirect all the traffic to Azure CDN.
 `shutter: false` will disable the shutter page and redirect all the traffic back to Azure Frontdoor.
+
+This is a self service request and does not require Platform Operations to create, approve or merge.
```

Comment:
Can you move line 47/48 to the top of file?

Ideally want it to b seen as soon as a user looks at page.